package.json

```diff
@@ -102,14 +86,16 @@
     "webpack-cli": "^4.7.2"
   },
   "workspaces": [
-    "samples/*",
-    "packages/*",
-    "e2e/*"
+    "samples/reactjs-todo",
+    "samples/todo-api",
+    "packages/javascript-sdk",
+    "e2e/autoscript-apps",
+    "e2e/autoscript-suites",
+    "e2e/mock-api",
+    "samples/central-login",
+    "samples/embedded-login"
   ],
   "dependencies": {
-    "core-js": "^3.19.2",
-    "nx": "^13.1.4",
-    "regenerator-runtime": "0.13.7",
-    "rxjs": "^7.4.0"
```

Comment:
I see these being removed in this PR, but not moved to to the package.json of the e2e apps. I did notice it in this PR: https://github.com/ForgeRock/forgerock-javascript-sdk/pull/184/files. I'm a bit worried about keeping tracking of all these dependencies.

Reply:
These commits are identical (I branched off that PR because I didn't wanna deal with the dependencies in a seperate branch).

https://github.com/ForgeRock/forgerock-javascript-sdk/blob/54933ba4e0803581cedf2cc3783e6c05f52ae413/e2e/autoscript-apps/package.json#L3 

They are moved here, were you looking elsewhere?

Reply:
Ah, sorry, must have missed that! Never mind then :)